### PR TITLE
📝  Update README with instructions

### PR DESCRIPTION
**Pull request by Robdel12**
## What is this?

This PR updates the README to contain setup instructions & docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,122 @@
...
+
+#### Installing `@percy/cli` & removing `@percy/agent`
+
+If you're coming from a pre-3.0 version of this package, make sure to install `@percy/cli` after
+upgrading to retain any existing scripts that reference the Percy CLI
+command. You will also want to uninstall `@percy/agent`, as it's been replaced
+by `@percy/cli`.
+
+```sh-session
+$ npm install --save-dev @percy/cli
```
Comment (wwilsman) [inline README.md:112]: Should we also add an uninstall snippet?

```sh-session
$ npm uninstall @percy/agent
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,122 @@
...
+[percy] Created build #1: https://percy.io/[your-project]
+[percy] Snapshot taken "Java example"
+[percy] Stopping percy...
+[percy] Finalized build #1: https://percy.io/[your-project]
+[percy] Done!
+```
+
+## Configuration
+
+`percy.snapshot(driver, name, [widths], minHeight, enableJavaScript, percyCSS)`
```
Comment (wwilsman) [inline README.md:75]: I think you want `widths[]` here. Unless you're trying to denote optional parameters, in which case you'll want to include the previous comma in the brackets and repeat for the remaining options

File: README.md (hunk 1 above)
Comment (Robdel12) [reply README.md:113]: Yea, for sure.

**Review by wwilsman — APPROVED** [review]
🎮

**Pull request merged**